### PR TITLE
Added RBAC for heapster in kubemark

### LINKED ISSUE #39952: Heapster in Kubemark doesn't work
Similar to issue #39245 .
Heapster seems to talk to the wrong apiserver (the one of the real cluster) instead of the kubemark apiserver.
To fix this, we need to replace the heapster image with a newer one (where this [change](https://github.com/kubernetes/heapster/pull/1468) has been incorporated).
One other (minor) thing is to bind the right role to heapster for RBAC.

cc @kubernetes/sig-scalability-misc @wojtek-t @gmarek

### PR DESCRIPTION
**Pull request by shyamjvs**
Fixes #39952 

cc @wojtek-t @gmarek @deads2k

### CODE REVIEW

**Comment by deads2k** [comment]
Role created here based on inspection: https://github.com/kubernetes/kubernetes/pull/40028

Heapster is relatively small, but rbac denies would be really useful for others.

File: test/kubemark/resources/manifests/addons/kubemark-rbac-bindings/heapster-binding.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+apiVersion: rbac.authorization.k8s.io/v1alpha1
+kind: ClusterRoleBinding
+metadata:
+  name: heapster-view-binding
+  labels:
+    kubernetes.io/cluster-service: "true"
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
```
Comment (wojtek-t) [inline test/kubemark/resources/manifests/addons/kubemark-rbac-bindings/heapster-binding.yaml]: @deads2k - is this correct? Shouldn't this be "system:heapster" ?

Comment (deads2k) [inline test/kubemark/resources/manifests/addons/kubemark-rbac-bindings/heapster-binding.yaml]: > @deads2k - is this correct? Shouldn't this be "system:heapster" ?

Yes it should say `system:heapster`

Comment (shyamjvs) [inline test/kubemark/resources/manifests/addons/kubemark-rbac-bindings/heapster-binding.yaml]: Now that #40028 is merged, changing this.

**Comment by shyamjvs** [comment]
@wojtek-t Does it look ok now?

File: test/kubemark/resources/manifests/addons/kubemark-rbac-bindings/heapster-binding.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# This is the role binding for the kubemark heapster.
+# TODO: Make this role weaker but still sufficiently powerful.
```
Comment (kargakis) [inline test/kubemark/resources/manifests/addons/kubemark-rbac-bindings/heapster-binding.yaml]: Does this TODO still hold true now that the heapster role is granted only the permissions it needs?

Comment (wojtek-t) [inline test/kubemark/resources/manifests/addons/kubemark-rbac-bindings/heapster-binding.yaml]: It's not - please remove

**Review by wojtek-t — COMMENTED** [review]
other than that, lgtm

File: test/kubemark/resources/manifests/addons/kubemark-rbac-bindings/heapster-binding.yaml (hunk 2 above)
Comment (shyamjvs) [inline test/kubemark/resources/manifests/addons/kubemark-rbac-bindings/heapster-binding.yaml]: Done. Sorry for the blooper.

**Comment by shyamjvs** [comment]
Adding lgtm based on @wojtek-t 's comment.

**Pull request merged**